packages/image/src/__test__/load.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { load } from "..";
+
+class MockImage {
+  public src = "";
+
+  public setAttribute(name: string, value: string) {
+    this[name] = value;
+  }
+
+  public constructor() {
+    MockImage.lastInstance = this;
+  }
+
+  public static lastInstance: MockImage;
+}
+
+describe("load", () => {
+  const originalImage = window.Image;
+
+  beforeAll(() => {
+    window.Image = MockImage as unknown as typeof Image;
+  });
+
+  afterAll(() => {
+    window.Image = originalImage;
+  });
+
+  it("should load defaultSrc when webpSrc is not provided", () => {
+    load([
+      {
+        defaultSrc: "./images/test.png",
+      },
+    ]);
+
+    expect(MockImage.lastInstance.src).toBe("./images/test.png");
+  });
+
+  it("should load webpSrc when provided", () => {
+    load([
+      {
+        defaultSrc: "./images/test.png",
+        webpSrc: "./images/test.webp",
+      },
+    ]);
+
+    expect(MockImage.lastInstance.src).toBe("./images/test.webp");
+  });
```

Comment:
How about make `load` return image

```suggestion
  it("should load webpSrc when provided", async () => {
    const loaded = await load([
      {
        defaultSrc: "./images/test.png",
        webpSrc: "./images/test.webp",
      },
    ]);

    expect(loaded.src).toBe("./images/test.webp");
  });
```

Reply:
how about this

```tsx
    const loaded = await load('./images/test', ['webp', 'png']); // first one is trying, but if not available webp, png is fallback
```

Reply:
I don't think it's right for the current PR to handle changing existing logic just to change a mock class in test code.

The current PR is created for code testing.

Reply:
this review is where logic code changes need to be made for test code

Reply:
https://github.com/fepack/image/pull/33#discussion_r1333391539

Reply:
If need to BREAKING CHANGE interface, I think before merging this Pull Request is best timing to change interface before add test code.

How about new Pull Request to change this load interface first if you agree my suggestion. What is your opinion

Reply:
```ts
export const load = (images: ImageSource[]) => {
  const loadedImages = [];

  for (const image of images) {
    const imageElement = new Image();
    imageElement.src = image.webpSrc ? image.webpSrc : image.defaultSrc;
    loadedImages.push(imageElement);
  }

  return loadedImages;
};
```

Do you think `load` should have a return? For example, at the code level, is the above format correct?

Reply:
How about making this comment as issue

Reply:
> How about making this comment as issue

#35